spectator/meter/counter.go

```diff
@@ -8,54 +13,39 @@ package spectator
 //
 // https://netflix.github.io/spectator/en/latest/intro/counter/
 type Counter struct {
-	count uint64
-	// Pointers need to be after counters to ensure 64-bit alignment. See
-	// note in atomicnum.go
-	id    *Id
+	id              *Id
+	writer          writer.Writer
```

Comment:
Is this going to keep a separate copy of the writer with each instantiation of the meter?

The way we did it in the other implementations was to have a single writer in the Registry that would be re-used by all meters.

Reply:
No, it'll be a pointer to the writer because `writer.Writer` is an interface so it'll by passed by reference.